error/src/lib.rs

```diff
@@ -19,162 +19,393 @@ use core::num::NonZeroU32;
+        CaliptraError::new_const(0x000b002a);
+    ///  RuntimeEntryPointUnaligned = 43,
+    pub const IMAGE_VERIFIER_ERR_RUNTIME_ENTRY_POINT_UNALIGNED: CaliptraError =
+        CaliptraError::new_const(0x000b002b);
+    /// RuntimeSvnGreaterThanMaxSupported = 44,
+    pub const IMAGE_VERIFIER_ERR_RUNTIME_SVN_GREATER_THAN_MAX_SUPPORTED: CaliptraError =
+        CaliptraError::new_const(0x000b002c);
+    /// RuntimeSvnLessThanMinSupported = 45,
+    pub const IMAGE_VERIFIER_ERR_RUNTIME_SVN_LESS_THAN_MIN_SUPPORTED: CaliptraError =
+        CaliptraError::new_const(0x000b002d);
+    /// RuntimeSvnLessThanFuse = 46,
+    pub const IMAGE_VERIFIER_ERR_RUNTIME_SVN_LESS_THAN_FUSE: CaliptraError =
+        CaliptraError::new_const(0x000b002e);
+    /// ImageLenMoreThanBundleSize = 47,
+    pub const IMAGE_VERIFIER_ERR_IMAGE_LEN_MORE_THAN_BUNDLE_SIZE: CaliptraError =
+        CaliptraError::new_const(0x000b002f);
+
+    /// Driver Error: LMS
+    pub const DRIVER_LMS_INVALID_LMS_ALGO_TYPE: CaliptraError =
+        CaliptraError::new_const(0x000c0001);
+    pub const DRIVER_LMS_INVALID_LMOTS_ALGO_TYPE: CaliptraError =
+        CaliptraError::new_const(0x000c0002);
+    pub const DRIVER_LMS_INVALID_WINTERNITS_PARAM: CaliptraError =
+        CaliptraError::new_const(0x000c0003);
+    pub const DRIVER_LMS_INVALID_PVALUE: CaliptraError = CaliptraError::new_const(0x000c0004);
+
+    pub const DRIVER_LMS_INVALID_HASH_WIDTH: CaliptraError = CaliptraError::new_const(0x000c0005);
+
+    pub const DRIVER_LMS_INVALID_TREE_HEIGHT: CaliptraError = CaliptraError::new_const(0x000c0006);
+
+    pub const DRIVER_LMS_INVALID_Q_VALUE: CaliptraError = CaliptraError::new_const(0x000c0007);
+
+    pub const DRIVER_LMS_INVALID_INDEX: CaliptraError = CaliptraError::new_const(0x000c0008);
+
+    pub const DRIVER_LMS_PATH_OUT_OF_BOUNDS: CaliptraError = CaliptraError::new_const(0x000c0009);
+
+    /// Instantiate = 1,
+    pub const DRIVER_CSRNG_INSTANTIATE: CaliptraError = CaliptraError::new_const(0x000d0001);
+
+    /// Uninstantiate = 2,
+    pub const DRIVER_CSRNG_UNINSTANTIATE: CaliptraError = CaliptraError::new_const(0x000d0002);
+    /// Reseed = 3,
+    pub const DRIVER_CSRNG_RESEED: CaliptraError = CaliptraError::new_const(0x000d0003);
+    /// Generate = 4,
+    pub const DRIVER_CSRNG_GENERATE: CaliptraError = CaliptraError::new_const(0x000d0004);
+    /// Update = 5,
+    pub const DRIVER_CSRNG_UPDATE: CaliptraError = CaliptraError::new_const(0x000d0005);
+
+    pub const RUNTIME_INTERNAL: CaliptraError = CaliptraError::new_const(0x000E0001);
+    pub const RUNTIME_UNIMPLEMENTED_COMMAND: CaliptraError = CaliptraError::new_const(0x000E0002);
+    pub const RUNTIME_INSUFFICIENT_MEMORY: CaliptraError = CaliptraError::new_const(0x000E0003);
+    pub const RUNTIME_ECDSA_VERIF_FAILED: CaliptraError = CaliptraError::new_const(0x000E0004);
+
+    pub const FMC_RTL_ALIAS_UNIMPLEMENTED: CaliptraError = CaliptraError::new_const(0x000F0001);
```

Comment:
Should this be `FMC_RT_ALIAS_UNIMPLEMENTED`?

Reply:
done!